pkg/manager/internal.go

```diff
@@ -554,6 +579,19 @@ func (cm *controllerManager) engageStopProcedure(stopComplete <-chan struct{}) e
 }
 
 func (cm *controllerManager) startLeaderElectionRunnables() error {
+	cm.logger.Info("Starting prestart hooks")
```

Comment:
Nit: "starting" makes it sound like hooks may run asynchronously
```suggestion
	cm.logger.Info("Running prestart hooks")
```